src/main/java/uk/gov/hmcts/divorce/caseworker/service/print/NoticeOfProceedingsPrinter.java

```diff
@@ -41,47 +58,160 @@ public void sendLetterToApplicant1(final CaseData caseData, final Long caseId) {
         final boolean applicant2NoticeOfProceedingsGenerated = isApplicant2Represented || isApplicant2Offline;
 
         final Letter noticeOfProceedingsLetter = applicant1NoticeOfProceedingsGenerated && applicant2NoticeOfProceedingsGenerated
-            ? letters.get(1)
-            : firstElement(letters);
+            ? lettersWithDocumentTypeNop.get(1)
```

Comment:
I am not sure of this logic ? Do we generate multiple NOP's and how do we know we need the second one ? I assume order is maintained ?

Reply:
worth adding a comment so it's easy to figure out thanks